.flowconfig

```diff
@@ -5,6 +5,8 @@
 .*/dist/module
 ./build-identity-iframe.js
 ./iframes/identity/identity.js
+.*/src/lib/tag-parsers/.*
```

Comment:
Why did you decided to ignore this folder?

Reply:
The parsers deal with looking up DOM HTml elements and working with HTMLTypes in flow is a PITA